mmd_tools/converters/mesh_converter.py

```
@@ -0,0 +1,619 @@
+                file_node = cmds.shadingNode(
+                    "file", asTexture=True, name=shader + "_texture"
+                )
+                # ファイルパスを設定
+                maya_utils.set_attribute(
+                    file_node, "fileTextureName", full_texture_path, "string"
+                )
+                # dx11ShaderのMainTextureに接続
+                try:
+                    cmds.connectAttr(
+                        file_node + ".outColor", shader + ".MainTexture", force=True
+                    )
```

Ruff (0.12.2)

Do not use bare `except`

(E722)